app/policies/observation_policy.rb

```diff
@@ -20,4 +20,8 @@ def can_show_proposal!
     policy = PolicyFinder.policy_for(@user, @observation.proposal)
     policy.can_show!
   end
+
+  def user_is_observer!
```

Comment:
should this be a `?` method rather than `!` -- usually expect to see `?` for methods that return true/false and `!` for methods that modify the objects they are called on